.gitignore

```diff
@@ -8,5 +8,7 @@ cdp-*.txt
 /chromedp.test
 /chromedp.test.exe
 
+.idea
```

Comment:
remove this; this should go in your global gitignore, not in all of your projects.